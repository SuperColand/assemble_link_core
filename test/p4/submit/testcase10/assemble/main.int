.data
0

.text
addiu $a0 $0 0
addiu $a0 $0 10403
jal prime
syscall
