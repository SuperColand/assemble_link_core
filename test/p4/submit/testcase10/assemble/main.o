24040000
240428a3
0c000c04
0000000c
00048021
24080001
1208000c
24080002
11100005
0208001b
00008810
12200007
25080001
08000c08
00044821
ac090000
24090001
ac090004
03e00008
00044821
ac090000
24090000
ac090004
03e00008
