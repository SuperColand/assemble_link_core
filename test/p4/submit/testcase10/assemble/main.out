.data
0

.text
24040000
240428a3
0c000000
0000000c

.symbol
0	main

.relocation
8	prime
